From: Marius Kintel <marius@kintel.net>
Date: Wed, 2 Dec 2015 17:56:18 -0500
Subject: Show correct QOpenGLWidget vs. QGLWidget in Library Info

---
 src/QGLView.cc | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/src/QGLView.cc b/src/QGLView.cc
index 91d8ca6..86b1790 100644
--- a/src/QGLView.cc
+++ b/src/QGLView.cc
@@ -113,7 +113,13 @@ std::string QGLView::getRendererInfo() const
   std::string glewinfo = glew_dump();
   std::string glextlist = glew_extensions_dump();
 	// Don't translate as translated text in the Library Info dialog is not wanted
-  return glewinfo + std::string("\nUsing QGLWidget\n\n") + glextlist;
+  return glewinfo + 
+#ifdef USE_QOPENGLWIDGET
+		std::string("\nUsing QOpenGLWidget\n\n")
+#else
+		std::string("\nUsing QGLWidget\n\n")
+#endif
+		+ glextlist;
 }
 
 #ifdef ENABLE_OPENCSG
